docs/NEWS

```diff
@@ -1,3 +1,44 @@
+Avahi 0.9, the 'Odrodzenie' release
+===============================
+
+The avahi 0.9 release is mostly bug fix release, including a set of vulnerabilities.
+No changes to API happened, which would be incompatible.
+
+New Features:
+- Temporary files are stored in runstatedir, /run by default #137
+
+Notable Changes:
+- Wide-area support using unicast DNS were disabled in default installation #577
+- Qt3 and Qt4 bindings are deprecated, only Qt5 remains supported (avahi-qt)
+- GTK 2.0 bindings are deprecated (avahi-ui)
+- Mono bindings are deprecated (avahi-sharp, avahi-ui-sharp)
+- HOWL compatibility layer is deprecated (avahi-compat-howl)
+- Deprecated bindings will be removed from avahi sources in following releases,
+  direct usage of DBus interface is recommended instead.
+- Changed path where DBus files are installed to $datadir #376
+
+Security Fixes:
+- CVE-2023-38473: Reachable assertion in avahi_alternative_host_name #451
+- CVE-2023-38472: Reachable assertion in avahi_rdata_parse #452
+- CVE-2023-38471: Reachable assertion in dbus_set_host_name #453
+- CVE-2023-38470: Reachable assertion in avahi_escape_label #454
+- CVE-2023-38469: Reachable assertion in avahi_dns_packet_append_record #455
+- CVE-2023-1981: avahi-daemon can be crashed via DBus #375
+- CVE-2021-26720: Debian service were executed as root
```

Comment:
It's Debian-specific and can be dropped.

Reply:
Okay, thought about excluding it also. Dropped CVE-2021-26720.